{"name": "test_login_phone_error_case", "status": "passed", "attachments": [{"name": "log", "source": "59324ae9-3dfc-40fc-871b-eb0af0b41d6c-attachment.txt", "type": "text/plain"}], "start": 1597290970857, "stop": 1597290972383, "uuid": "19f30d95-f15e-4f45-a2c3-ed71c2a2e4fc", "historyId": "210797769f28844d15197db9e5f676ed", "testCaseId": "7bcb06678aeec0849c5a6efc7c1a2546", "fullName": "case.login.test_login_case.TestLoginCase#test_login_phone_error_case", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "case.login"}, {"name": "suite", "value": "test_login_case"}, {"name": "subSuite", "value": "TestLoginCase"}, {"name": "host", "value": "FYYX-2019VXOSDQ"}, {"name": "thread", "value": "11280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.login.test_login_case"}]}